client/src/App.js

```diff
@@ -36,8 +46,40 @@ class App extends React.Component {
+                                    <h1>
+                                        Welcome to Monuments + Memorials
+                                    </h1>
+                                    <div>
+                                        <img className='homePageImage' src={process.env.PUBLIC_URL + '/home_page_image.jpg'}/>
+                                    </div>
+                                    <div>
+                                        <div className='about-information-container'>
+                                            <p>
+                                                Thank you for visiting <span className='font-italic'>Monuments + Memorials</span>, a crowd-sourced initiative developed at Rochester Institute of Technology.
+                                                Begun in the spring 2019 with data collection and launched as a site in fall 2019, <span className='font-italic'>Monuments + Memorials</span> is a long-term
+                                                look at monuments, memorials, and memory in the US. To view monuments and memorials, {mapNavLink}. To enter
+                                                data, click here.
+                                                <br/>
+                                                <br/>
+                                                Our goal is to map every monument or memorial in the United States and its inhabited territories (American Samoa, Guam,
+                                                Puerto Rico, the North Mariana Islands, and U.S. Virgin Islands). We invite anyone to contribute information and an
+                                                image of monuments and memorials in these areas.
+                                                <br/>
+                                                <br/>
+                                                Students from K-12 and higher education, as well as community groups and other organizations, are invited to join us
+                                                in documenting our monuments and memorials. If you would like to join our team as a project partner and collect data
+                                                from a defined area (such as a city, town, or county), please let us know. We are looking for partners across the
+                                                United States.
+                                                <br/>
+                                                <br/>
+                                                If you have any questions, please contact the project coordinator, <b>Juilee Decker jdgsh@rit.edu</b>.
+                                            </p>
+                                        </div>
+                                    </div>
+                                    <div>
+                                        <SuggestChanges/>
+                                    </div>
```

Comment:
I think this entire block that you added should be it's own Page Component. That way if we ever end up adding more information to it or need to make an API request because it contains dynamic data, it's easy to do.

Reply:
I didn't mean to make the `SuggestChanges` component into a Page. Now we have 2 components that do the same thing.

I meant all of the content you have in `App.js` should be made into its own Page component.

Reply:
Specifically, what you have between lines 49 and 81.

Reply:
ok, that makes way more sense. Sorr for this misunderstanding